internal/jobservice/repository/subscribe_table.go

```diff
@@ -0,0 +1,12 @@
+package repository
+
+import "time"
+
+type SubscribeTable struct {
+	subscribedJobSet     string
+	lastRequestTimeStamp int64
+}
+
+func NewSubscribeTable(jobSetId string) *SubscribeTable {
```

Comment:
so generally `jobSet` isn't the primary key, rather `(queue, jobset`) is the primary key.  I'm not sure if your concatting the two somewhere and calling the result `jobset`, but even if so I'd probably recommend keeping them separate.

Note that this comment applies to a bunch of places where only `JobSet` is used as the PK.

Reply:
Good call out!  I was thinking job-set can be the PK but I will fix this.

Reply:
Alright I fixed this now too.